testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetITSupport.java

```diff
@@ -24,19 +24,31 @@
 import com.sonatype.nexus.repository.nuget.internal.NugetHostedRecipe;
 import com.sonatype.nexus.repository.nuget.odata.FeedSplicer;
 import com.sonatype.nexus.repository.nuget.odata.ODataConsumer;
+import com.sonatype.nexus.repository.nuget.security.NugetApiKey;
+import com.sonatype.nexus.repository.nuget.security.NugetApiKeyStore;
```

Comment:
Any reason we can't make these two types public while keeping the implementation types internal?